students/BRACH_Jakub/lesson_01_basics/L01P19_ClockFace.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+import math
+
+print('Enter the hour hand angle')
+hour_angle = float(input())
+
+
+
+minute_angle = (hour_angle * 12) % 360
+
+
+print('minute angle will be {0:f} deg.'.format(minute_angle))
+
```

Comment:
Read statement more carefully:
Determine the angle (in degrees) of the **hour** hand 

Please check test scenarios at:
https://snakify.org/lessons/integer_float_numbers/problems/clock_face_1/

Reply:
I answered wrong problem. Mea culpa.